site/layouts/404.html

```diff
@@ -0,0 +1,27 @@
+{{ define "siteHeader" }}
+
+{{ template "_default/site-header.html" . }}
+
+{{ end }}
+
+{{ define "content"}}
+<p> Whoops! It looks like we weren't able to find that page.<p>
+
+
+<ul>
+  <li> <a href="/docs/latest">Documentation</a></li>
+  <li> <a href="/plugins">Plugins</a></li>
+  <li> <a href="/blog">Blog</a></li>
+  <li> <a href="/community">Community</a></li>
+  <li> <a href="/resources">Resources</a></li>
+</ul>
+
+<p> If you can't find what you are looking for, try asking a question in our <a href="https://kubernetes.slack.com/messages/velero">slack channel</a> or <a href="{{.Site.Params.gh_repo}}">filing an issue.</a><p>
```

Comment:
@a-mccarthy would you please make this link point to https://github.com/vmware-tanzu/velero/issues directly?

Reply:
@carlisia updated! thanks for the review